manim/scene/scene.py

```diff
@@ -116,9 +122,37 @@ def tear_down(self):
         pass
 
     def construct(self):
-        """
-        The primary method for constructing (i.e adding content to)
-        the Scene.
+        """Add content to the Scene.
+
+        From within :meth:`Scene.construct`, display mobjects on screen by calling
+        :meth:`Scene.add` and remove them from screen by calling :meth:`Scene.remove`.
+        All mobjects currently on screen are kept in :attr:`Scene.mobjects`.  Play
+        animations by calling :meth:`Scene.play`.
+
+        Notes
+        -----
+        Initialization code should go in :meth:`Scene.setup`.  Termination code should
+        go in :meth:`Scene.tear_down`.
+
+        Examples
+        --------
+        A typical manim script includes a class derived from :class:`Scene` with an
+        overriden :meth:`Scene.contruct` method:
+
+        .. code-block:: python
+
+            class MyScene(Scene):
+                def construct(self):
+                    t = Text("Hello World!")
+                    self.add(t)
+                    self.play(Write(t))
```

Comment:
self.add(t)
                    self.play(Write(t))
I think one of both can be removed

Reply:
I believe it's a lot better to try to encourage that every mobject is added before it is animated, even if this is not necessary.

Reply:
I dont think so, as e.g. this script:
```python
class ABC(Scene):
    def construct(self):
        t = Text("Hello World!")
        self.add(t)
        self.wait()
        self.play(Write(t))
```
would not make sense then.
And `self.play(Write(t))` also adds the object to the scene, as you can see in this example
```python
class ABC(Scene):
    def construct(self):
        t = Text("Hello World!")
        self.wait()
        self.play(Write(t))
        self.remove(self.mobjects[-1])
        self.wait()
```